cms/static/cms/js/modules/cms.plugins.js

```diff
@@ -673,8 +733,18 @@ var Plugin = new Class({
 
         // set new setting on instance and plugin data
         this.options = settings;
-        plugin.data('settings', settings);
-        draggable.data('settings', settings);
+        if (plugin.length) {
+            var index = plugin.data('cms').findIndex(function (pluginData) {
```

Comment:
not really. we don't enforce 'vars-on-top' in cms, too much stuff would need updating